packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -210,8 +211,20 @@ abstract class OAuth2Client {
 
   /**
    * Invokes the OIDC end session endpoint.
+   * Can result in a redirect to `/signoff` if using PingOne
+   * It's best to explicitly provide the logout redirect URL in options
+   *
+   * @function endSession - call authorization server to end associated session
+   * @param options {LogoutOptions} - an extension of ConfigOptions, but with two additional props
+   * @param options.logoutRedirectUri {string} - the URL you want the AS to redirect to after signout
+   * @param options.redirect {boolean} - to explicitly deactivate redirect, pass `false`
```

Comment:
I've moved the `signoutUrl` from the central config to the method options, keeping the central config clean.